samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AssessmentSettingsBean.java

```diff
@@ -1773,7 +1774,7 @@ public void saveEditedExtendedTime() {
       Integer oldIndex = null;
       // is there a more efficient way?
       for(ExtendedTime entry : extendedTimes) {
-        if(entry.getId().equals(this.extendedTime.getId())) {
+        if(ObjectUtils.equals(entry.getId(), this.extendedTime.getId())) {
```

Comment:
Yeah, getId() was null so having a ObjectUtils.equals is a lot safer for both being nulls. 

The only thing I wasn't sure about was if we wanted it to break if they were both null, but it fixed the but so I was happy with it. 

Ah thanks for the deprecation tip it's built right into Java now! I can change that.

this method has been replaced by java.util.Objects.equals(Object, Object) in Java 7